.github/workflows/tag-release.yml

```
@@ -0,0 +1,35 @@
+      - run: npm ci
+      - run: npm run pack
+      - uses: actions/upload-artifact@v3
+        with:
+          name: extension
+          path: web-ext-artifacts/*.zip
+
+  release:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/download-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/download-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)